docs/release_notes/0.4.0.md

```diff
@@ -1,10 +1,24 @@
 # Release 0.4.0
 
 ## Features
-- ...
+
+Includes all features of the previous version, plus:
```

Comment:
How about we just rename the heading to "New Features"? This line is quite redundant.

Reply:
It is from a suggestion from a user in a different PR earlier this week.